src/connectivity.jl

```diff
@@ -263,7 +285,7 @@ function strongly_connected_components end
                                 break
                             end
                         end
-                        push!(components, reverse(component))
+                        push!(components, reverse!(component))
```

Comment:
This is a bit of a hack. It relies on the fact that `reverse!` returns the modified vector, but strictly speaking, it doesn't HAVE to since it's a mutating function. Better to call `reverse!` component on the line before and use `component` here.

Reply:
Changed.